kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py

```diff
@@ -397,16 +470,14 @@
             "firefox",
             "mobile",
             "ios",
-            "firefox-private-network-vpn",
-            "relay",
-            "monitor",
-            "pocket",
-            "focus-firefox",
-            "firefox-enterprise",
-            "thunderbird",
-            "mozilla-account",
         ],
         "subtopics": [
+            {
+                "title": "Data removal",
+                "slug": "data-removal",
+                "description": "Remove your data from Mozilla’s products.",
+                "products": ["monitor"],
+            },
             {
                 "title": "Tracking protection",
```

Comment:
As per the taxonomy doc, for the Tracking Protection tier 2:
* we are missing the Relay & Pocket products. 
* both thunderbird & thunderbird-android should not be included.